src/test/java/com/ddudu/user/service/FollowingServiceTest.java

```diff
@@ -227,6 +229,23 @@ class 팔로잉_수정_테스트 {
       ThrowingCallable updateStatus = () -> followingService.updateStatus(
           invalidId, invalidId, request);
 
+      // then
+      assertThatExceptionOfType(DataNotFoundException.class).isThrownBy(updateStatus)
+          .withMessage(UserErrorCode.ID_NOT_EXISTING.getMessage());
+    }
+
+    @Test
+    void 존재하지_않는_아이디면_실패한다() {
+      // given
+      User follower = createUser();
+      long invalidId = faker.random()
+          .nextLong();
```

Comment:
```suggestion
          .nextLong(Long.MAX_VALUE);
```

여기도..! 근데 우선 넘어가도 될 것 같네요..!